src/main/java/org/motechproject/mots/dto/CommunityCreationDto.java

```diff
@@ -15,19 +17,13 @@
 
   @Getter
   @Setter
-  @NotBlank(message = ValidationMessages.EMPTY_LOCATION_NAME)
+  @NotBlank(message = ValidationMessages.EMPTY)
   private String name;
 
   @Setter
   @Getter
-  private String districtId;
-
-  @Setter
-  @Getter
-  private String chiefdomId;
-
-  @Setter
-  @Getter
-  @NotBlank(message = ValidationMessages.EMPTY_FACILITY_TYPE)
+  @FacilityExistence
+  @Uuid
+  @NotEmpty(message = ValidationMessages.EMPTY)
```

Comment:
Let's bring back full validation message. The generic validation will be added later on

Reply:
Corrected and added missing validations.

Reply:
it's still using ValidationMessages.EMPTY as opposed to full message. is that on purpose?